Gems/Pointcloud/Assets/Shaders/Pointclouds/Pointclouds.shader

```diff
@@ -0,0 +1,28 @@
+{
+  "Source": "Pointclouds.azsl",
+  "DepthStencilState": {
+    "Depth": {
+      "Enable": true,
+      "CompareFunc": "GreaterEqual"
+    }
+  },
+  "GlobalTargetBlendState" : {
+      "Enable" : false,
+      "BlendSource" : "Zero",
+      "BlendDest" : "One",
+      "BlendOp" : "Add"
+  },
+  "DrawList": "auxgeom",
```

Comment:
Using auxgeom draw list is causing manipulators to flicker.
[Screencast from 07-30-2024 12:05:44 PM.webm](https://github.com/user-attachments/assets/00a7bbc1-7ac3-46d4-bedd-784af1167b71)

Reply:
I will map it to issue.